RELEASE.md

```diff
@@ -1,3 +1,16 @@
+## Release 1.7.2
+### Major Features and Improvements
+> FederatedML
+* New batch strategy in coordinated Hetero LR: support masked batch data and batch shuffle
+* Model inference protection enhancement for Hetero SecureBoost with FED-EINI algorithm
+* Hetero SecureBoost supports split feature importance on host side, disables gain feature importance
+* Offline SBT Feature transform component 
+
+> Bug-Fix
+* Fixed Bug for HeteroPearson with changing default q_field value for spdz
+* Fix Data Transform's schema label name setting problem when `with_label` is False
+
```

Comment:
examples modified a very large number of files, is it necessary to specify in the release?

Reply:
"Add testing examples for new algorithm features, and delete deprecated params in algorithm examples"  is added to release.md and the PR descriptions